src/Service/NotificationService.php

```diff
@@ -31,9 +31,6 @@ class NotificationService implements NotificationServiceInterface
 {
     private ?string $validatedAdminNotificationEmail;
 
-    /**
-     * @param non-empty-string $bindNotificationTimezone
-     */
     public function __construct(
```

Comment:
Consider keeping the `@param non-empty-string $bindNotificationTimezone` annotation to tell programmers and the code analysis that a non-empty string is required (in addition to the actual checks in code).